src/server/game/Entities/Object/ObjectGuid.h

```diff
@@ -33,31 +33,39 @@ enum TypeID
     TYPEID_OBJECT        = 0,
     TYPEID_ITEM          = 1,
     TYPEID_CONTAINER     = 2,
-    TYPEID_UNIT          = 3,
-    TYPEID_PLAYER        = 4,
-    TYPEID_GAMEOBJECT    = 5,
-    TYPEID_DYNAMICOBJECT = 6,
-    TYPEID_CORPSE        = 7,
-    TYPEID_AREATRIGGER   = 8,
-    TYPEID_SCENEOBJECT   = 9,
-    TYPEID_CONVERSATION  = 10
+    TYPEID_AZERITE_EMPOWERED_ITEM = 3,
+    TYPEID_AZERITE_ITEM  = 4,
```

Comment:
Cosmetic, but it would be nice if you could align the enum indents above and below `TYPEID_AZERITE_EMPOWERED_ITEM = 3,` so the enum values are in the same column.